addons/account/tests/test_product.py

```diff
@@ -52,3 +57,10 @@ def test_multi_company_product_tax(self):
             'taxes_id': self.company_data['company'].account_sale_tax_id.ids,
             'supplier_taxes_id': self.company_data['company'].account_purchase_tax_id.ids,
         }])
+
+    def test_account_manager_user_can_create_product(self):
+        """Test that a user with group_account_manager can create a product."""
+        product = self.env['product.product'].with_user(self.account_manager_user).create({
+            'name': 'Test Accountant', 'type': 'product', 'list_price': 50.0,
+        })
+        self.assertTrue(product)
```

Comment:
```suggestion
        self.assertTrue(product)
        
```
We just need a newline at the end of the file :smile:

Reply:
Indeed, thank you !